samples/posix/eventfd/README.rst

```diff
@@ -0,0 +1,45 @@
+.. _posix-eventfd-sample:
+
+POSIX eventfd()
+###############
+
+Overview
+********
+
+This sample application demonstrates using the POSIX eventfd() function to create a file descriptor,
+which can be used for event notification. The returned file descriptor is used with write/read calls
+and write/read values are output to the console.
+
+Building and Running
+********************
+
+This project outputs to the console.  It can be built and executed on QEMU as follows:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/posix/eventfd
+   :host-os: unix
+   :board: qemu_x86
+   :goals: run
+   :compact:
+
+For comparison, to build directly for your host OS if it is POSIX compliant (for ex. Linux):
+
+.. code-block:: console
+
+   make -f samples/posix/eventfd/Makefile.host
+
+The make output file will be located in samples/posix/eventfd.
```

Comment:
non blocking: kind of not ideal that this adds a new untracked file to the git repo, but I guess it was already like that before.